### PR TITLE
Update_time_fa

### PR DESCRIPTION
**Pull request by cristinapunti**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def find_version(*file_paths):
 with open(path.join(here, 'README.md'), encoding='utf-8') as readme_file:
     long_description = readme_file.read()
 
-VERSION = find_version('face_alignment', '__init__.py')
+#VERSION = find_version('face_alignment', '__init__.py')
+VERSION = '2.0.0' #TODO Remove hardcoded value
```
Comment (jescur) [inline setup.py]: check what we do with that

**Review by ibrarmalik — APPROVED** [review]
Nice!

File: face_alignment/api.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class NetworkSize(IntEnum):
 
 
 class FaceAlignment:
-    def __init__(self, landmarks_type, face_align_model_path, depth_pred_model_path, network_size=NetworkSize.LARGE,
+    def __init__(self, landmarks_type, face_align_model_path, depth_pred_model_path=None, network_size=NetworkSize.LARGE,
```
Comment (ibrarmalik) [inline face_alignment/api.py:30]: Does this work when `depth_pred_model_path` is None?

Comment (cristinapunti) [reply face_alignment/api.py:30]: Yes, by using this 'None' variable, we can keep the structure in case we need it in the future. Now it is not used!

Comment (ibrarmalik) [reply face_alignment/api.py:30]: Got it ✌️

**Pull request merged**